docs/apps/authentik.md

```diff
@@ -0,0 +1,88 @@
+---
+hide:
+  - tags
+tags:
+  - authentik
+---
+
+# Authentik
+
+## What is it?
+
+[Authentik](https://goauthentik.io/) is an IdP (Identity Provider) and SSO (single sign on) that is built with security at the forefront of every piece of code, every feature, with an emphasis on flexibility and versatility. It supports all of the major providers, such as OAuth2, SAML, LDAP, and SCIM, so you can pick the protocol that you need for each application.
+
+| Details     |             |             |             |
+|-------------|-------------|-------------|-------------|
+| [:material-home: Project home](https://goauthentik.io/){: .header-icons } | [:octicons-link-16: Docs](https://docs.goauthentik.io/docs/){: .header-icons } | [:octicons-mark-github-16: Github](https://github.com/goauthentik/authentik){: .header-icons } | [:material-docker: Docker](https://hub.docker.com/r/beryju/authentik){: .header-icons }|
+
+### 1. Installation
+
+``` shell
+
+sb install authentik
+
+```
+
+### 2. URL
+
+- To access Authentik, visit `https://authentik._yourdomain.com_`
```

Comment:
This is not the correct default subdomain - it's auth.

Reply:
Omg I know why didn't I change that. I even made note of it. Thanks!